README.md

```diff
@@ -37,10 +37,25 @@ unzip 2019-06-20-raspbian-buster-lite.zip && cd ..
 
 Bootstrap the demo rootfs overlay that is configured to connect to
 https://hosted.mender.io with polling intervals set appropriately for
-demonstration purposes:
+demonstration purposes.  There are three scripts here to support
+the Mender demo server, the Mender production server, and hosted
+Mender.
 
 ```
-./scripts/bootstrap-rootfs-overlay-demo.sh \
+./scripts/bootstrap-rootfs-overlay-demo-server.sh \
+    --output-dir ${PWD}/rootfs_overlay_demo \
+    --server-ip 192.168.1.1
+```
+
+```
+./scripts/bootstrap-rootfs-overlay-production-server.sh \
+    --output-dir ${PWD}/rootfs_overlay_demo \
+    --server-url https://foobar.mender.io \
+    [ --server-cert ~/server.crt ]
+```
```

Comment:
I was really hoping to avoid covering these variations here ( and providing multiple scripts), and instead I was hoping that we would cover this in docs.mender.io instead. How rootfs overlay is utilized to apply configurations, and docs.mender.io should cover (in general) the different server configuration options instead of us duplicating it everywhere.

We are also moving away of providing multiple configurations, in other parts of the docs, e.g https://docs.mender.io/2.1/client-configuration/installing. This only covers Hosted Mender as well, and maybe this is the appropriate location to update on how to handle server variants.

We do have https://tracker.mender.io/browse/MEN-2786 to complete, and once this is finished I would think these parts would crystallize.

Reply:
I think it is ok if you add the scripts for now. But I would omit the update of the README mentioning all variants.

Reply:
Again, why?  The main effort lately has been on ease of use and requiring extra steps through the docs complicates that.  We support 3 main methods of servers at the moment so it seems like this should mirror that.